routes/common/base.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Routes::Common::Base
+  module AppMode
+    API = "api"
+    WEB = "web"
```

Comment:
Interesting idea... I can think about few places where we can use the class. But we already have the class instances in the `@app`, which is generally more useful as we can use instance methods. Otherwise, I'd make them class.

I converted them to symbols.